### PR TITLE
SpaceController & SpaceForm : Can now add spaces without slot entity …

### PR DESCRIPTION
**Pull request by MarvinCrepin**
…(simple version)

### CODE REVIEW

File: src/Controller/SpaceController.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    public function new(Request $request): Response
+    {
+        $spaceForm = new Space();
+
+        $form = $this->createForm(SpaceType::class, $spaceForm);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $entityManager = $this->getDoctrine()->getManager();
```
Comment (rocambille) [inline src/Controller/SpaceController.php]: follow good practices and use dependency injection. in fact, just use a make:crud : that will generate the right code

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/Controller/SpaceController.php (hunk 1 above)
Comment (MarvinCrepin) [reply src/Controller/SpaceController.php]: done

**Comment by rocambille** [comment]
image

convert parameter 2 into string

**Comment by MarvinCrepin** [comment]
> image
> 
> convert parameter 2 into string

done

**Pull request merged**